package.json

```diff
@@ -6,9 +6,9 @@
     "node": ">=8.0.0"
   },
   "dependencies": {
-    "@enact/core": "^4.5.0-alpha.1",
+    "@enact/core": "^4.5.0",
     "@enact/docs-utils": "^0.3.7",
-    "@enact/ui": "^4.5.0-alpha.1",
+    "@enact/ui": "^4.5.0",
```

Comment:
this might be a duplicate of WRO-8682, but I'll approve and let the next reviewer decide